### ISSUE TITLE
Fix tab order

### ISSUE THREAD

**Issue by FRosner** [FRosner/DataGenerator#218]
Tab order in the main frame should be:
- Add Feature
- Feature Definition Area
- Edit Feature
- Remove Feature
- #Instances
- Export File Chooser
- Instance IDs
- Feature Names
- Generate Data / Abort Generation
- Log
